rules/62-arduino.rules

```diff
@@ -0,0 +1,2 @@
+# Make USB devices created for communicating with Arduino devices accessible by any user
+KERNEL=="ttyACM[0-9]*", MODE="0666"
```

Comment:
The original bug was misleading here - we kind of want to get any serial device, not just ttyACM ones. It's kind of arbitrary/hardware dependent whether you get ttyUSB, ttyACM or ttyS - so we should go closer to the "dialout" rule with something like `KERNEL=="tty[A-Z]*[0-9]"` I think.